### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by chraibi**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+m.chraibi@fz-juelich.de.
```
Comment (schroedtert) [inline CODE_OF_CONDUCT.md:63]: Maybe you want to protect your mail address against any website crawlers. Just as a hint, it's up to you.

**Review by schroedtert — APPROVED** [review]
For me this is fine. 

Is there a specific reason you chose this Code of Conduct?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (chraibi) [reply CODE_OF_CONDUCT.md]: Done!

**Comment by chraibi** [comment]
> For me this is fine.
> 
> Is there a specific reason you chose this Code of Conduct?

It was suggested by GitHub. Kinda  a template.

**Review by schroedtert — APPROVED** [review]

**Pull request merged**